src/painting/painter.rs

```diff
@@ -123,7 +126,7 @@ impl Painter {
 
     /// Returns the available lines from the prompt down
     pub fn remaining_lines(&self) -> u16 {
-        self.screen_height().saturating_sub(self.prompt_start_row)
+        self.screen_height().saturating_sub(self.prompt_height)
```

Comment:
Hmm, should this also subtract the start row, to account for the prompt not being at the top of the terminal?
```suggestion
        self.screen_height().saturating_sub(self.prompt_start_row).saturating_sub(self.prompt_height)
```